runtime/redirector/forwarders.m4

```diff
@@ -390,6 +390,8 @@ _IF([JAVA_SPEC_VERSION >= 17],
 	[_X(JVM_DumpClassListToFile, JNICALL, false, void, JNIEnv *env, jstring str)])
 _IF([JAVA_SPEC_VERSION >= 17],
 	[_X(JVM_DumpDynamicArchive, JNICALL, false, void, JNIEnv *env, jstring str)])
+_IF([JAVA_SPEC_VERSION >= 17],
+	[_X(JVM_GetProperties, JNICALL, false, jobjectarray, JNIEnv *env)])
```

Comment:
This needs to use `jobjectArray` (with a capital A).

Reply:
Updated.